lms/envs/aws.py

```diff
@@ -171,6 +171,10 @@
 for feature, value in ENV_FEATURES.items():
     FEATURES[feature] = value
 
+# Backward compatibility for deprecated feature names
+if 'ENABLE_S3_GRADE_DOWNLOADS' in FEATURES:
+    FEATURES['ENABLE_GRADE_DOWNLOADS'] = FEATURES['ENABLE_S3_GRADE_DOWNLOADS']
```

Comment:
What happens if someone has both `ENABLE_GRADE_DOWNLOADS` and `ENABLE_S3_GRADE_DOWNLOADS` defined? Which takes precedence?